CHANGELOG.md

```diff
@@ -237,7 +310,7 @@ The purchase modal will not show correctly for Amazon in these environments.
 ## 3.4.5
 
 - Bumped purchases-android to 4.3.1 [Changelog here](https://github.com/RevenueCat/purchases-android/releases/4.3.1),
-which fixes canMakePayments not returning (see related issue: https://github.com/RevenueCat/purchases-unity/issues/61)
+which fixes canMakePayments not returning (see related issue: https://github.com/RevenueCat/purchases-flutter/issues/61)
```

Comment:
Hmm just want to confirm this is intentional?

Reply:
Reverting. Actually neither https://github.com/RevenueCat/purchases-unity/issues/61 nor https://github.com/RevenueCat/purchases-flutter/issues/61 seem to be the correct one and they look unrelated to  "canMakePayments not returning"